### DISCUSSION TITLE
Future Maui, Full Capabilities in Designs and Menu Toolbar Animations

### DISCUSSION THREAD

**Discussion by nextcodelab** [dotnet/maui#3619] [Ideas]
This is just an inspiration that maybe you can do all the kinds of design capabilities in .Net Maui in the future.
     There are thousands perfect beautiful designs created in flutter, when you search in internet. Maybe because flutter is not native UI dependent. 
     Like if you want to change the specific design in element, you can't do that if other platform not support it, that is the always the case in xamarin, I hope not in Maui.
Here are the  some of thousands beautiful design in flutter, this is because of its unlimited design capabilities and animations.

[https://flutterawesome.com/tag/templates/](https://flutterawesome.com/tag/templates/)
[CodeFlutters](https://codecanyon.net/category/mobile/flutter?sort=sales&gclid=Cj0KCQiA-qGNBhD3ARIsAO_o7ymL8akJTXPsaZV1-HfUfozqaQS3AKqS5sJOSnPqGrVox1ckmBEsIZUaAs9WEALw_wcB#content)

I KNOW AND COMPORTABLE TO USE C# THAN DART.

**Comment by saint4eva** [comment]
https://github.com/jsuarezruiz/xamarin-forms-goodlooking-UI

https://snppts.dev/category/5

https://www.syncfusion.com/essential-xamarin-ui-kit
